main/templates/main/includes/navbar.html

```diff
@@ -13,3 +16,19 @@
     </ul>
   </div>
 </nav>
+
+<!-- Mobile Navigation -->
+<ul class="sidenav" id="mobile-demo">
+  <li><a href="sass.html">Sass</a></li>
+  <li><a href="badges.html">Components</a></li>
+  <li><a href="collapsible.html">Javascript</a></li>
+  <li><a href="mobile.html">Mobile</a></li>
+</ul>
```

Comment:
Change these accordingly

Reply:
Arre yes, yeh maine kya kiya

Reply:
I've made the changes accordingly... thanks for your contribution.
I'm closing this request.